tests/performance-test/deploy/config/test-configs.yml

```diff
@@ -0,0 +1,25 @@
+- metadata:
+    name: SAF Performance Test 1
+  spec:
+    value-lists: 10000
+    hosts: 5000
+    plugins: 100
+    interval: 1
+    length: 20
+    queries:
+      - rate(sa_collectd_total_amqp_processed_message_count[10s])
+      - sa_collectd_cpu_total
```

Comment:
Can you change this to another metric? The `sa_collectd_cpu_total` I'm pretty sure only shows you the total number of available cores, which probably isn't what you want here. I'd suggest maybe listing a 2 or 3 extra metrics. You have the number of processed message counts. It would be useful to also see the actual CPU load, the memory load, and maybe the disk IO or something like that.

If you have a running environment (I just tore all mine down) we can look at the list and pick 2-3 of them that make sense here.

Reply:
Yeah I can add some better defaults - I really just have them there as an example. What do you think would be more useful default queries?